dynatrace/src/main/java/com/dynatrace/opentelemetry/metric/DynatraceMetricExporter.java

```diff
@@ -61,6 +75,7 @@ public static DynatraceMetricExporter getDefault() {
     try {
       builder
           .setUrl(new URL("http://127.0.0.1:14499/metrics/ingest"))
+          .setPrefix("otel.java")
```

Comment:
```suggestion
```
It would be surprising to users if their metric name would change per default.
This also not consistent with out other exporters.